src/components/ListingDetail/ListingDetail.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <div v-if="isLoading === true">
```

Comment:
You don't need use `=== true` because the variable `isLoading` is boolean, the conditional is implicit

Reply:
True kkk, thanks!